client/js/Core/componentsandbox.js

```diff
@@ -265,12 +265,16 @@ SCWeb.core.ComponentSandbox.prototype.updateContent = async function (contentTyp
     var self = this;
 
     if (this.is_struct && this.eventStructUpdate) {
+        const maxNumberOfTriplets = 100;
```

Comment:
I think we don't need this limitation because it's timely solution. We can wait for right solution

Reply:
let's keep this timely solution, it's really useful, maybe we can create issue?

Reply:
Ok, let's find optimal limitation and create issue